terraform/storage/main.tf

```diff
@@ -0,0 +1,52 @@
+# Create a service running on fargate with a task definition and service definition
+terraform {
+  backend "s3" {
+    # Backend is selected using terraform init -backend-config=path/to/backend-<env>.tfbackend
+    # bucket         = "sdp-dev-tf-state"
+    # key            = "sdp-sandbox-tech-audit-tool-api-lambda/terraform.tfstate"
+    # region         = "eu-west-2"
+    # dynamodb_table = "terraform-state-lock"
+  }
+
+}
+
+# Remove the ECR repository creation since it already exists
```

Comment:
Reiterating what Steve said, why is the ECR stuff needed here?

Reply:
Removed.